### PR TITLE
Prs5276 helena corr mk

### PR DESCRIPTION
**Pull request by MarcinKrawczuk**

### CODE REVIEW

File: PRS5001-6000/PRS5276Helena.xml [hunk 1]
```diff
@@ -32,26 +32,35 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+               <persName xml:lang="gez" corresp="#n1">እሌኒ፡</persName>
+               <persName xml:lang="gez" type="normalized" corresp="#n1">ʾƎleni</persName>
+               <occupation type="political">empress</occupation>
+               <birth notBefore="0248" notAfter="0250">248-250</birth>
+               <death when="0328">328</death>
+               <nationality type="ByzantineEmpire"></nationality>
             </person>
+            <listRelation>
+              <relation name="snap:MotherOf" active="PRS5276Helena" passive="PRS3175Constant"/>
+            </listRelation>
```
Comment (eu-genia) [inline PRS5001-6000/PRS5276Helena.xml:65]: maybe also add subject_of relation to NAR0145SenHelena
btw, could LIT6171HolyCross be possibly doublet to LIT1764Legend ?

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: PRS5001-6000/PRS5276Helena.xml [hunk 2]
```diff
@@ -55,7 +55,6 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                <occupation type="political">empress</occupation>
                <birth notBefore="0248" notAfter="0250">248-250</birth>
                <death when="0328">328</death>
-               <nationality type="ByzantineEmpire"></nationality>
                <floruit>Mother of emperor Constantine. In Ethiopian tradition she features in various texts related to
                the finding of the Holy Cross.</floruit>
```
Comment (eu-genia) [inline PRS5001-6000/PRS5276Helena.xml:59]: I would annotate Holy Cross `<ref type="authFile" corresp="TrueCross">Holy Cross</ref>`

File: PRS5001-6000/PRS5276Helena.xml (hunk 1 above)
Comment (MarcinKrawczuk) [reply PRS5001-6000/PRS5276Helena.xml:65]: Nobody has ever explained to me what NAR is so I'm not using it:( As for the doublet, we had this discussion https://github.com/BetaMasaheft/Documentation/issues/1619 There is article by Witold

File: PRS5001-6000/PRS5276Helena.xml (hunk 2 above)
Comment (MarcinKrawczuk) [reply PRS5001-6000/PRS5276Helena.xml:59]: Great idea, I didn't know it was possible

**Review by eu-genia — APPROVED** [review]

**Pull request merged**